build.gradle

```diff
@@ -156,6 +156,9 @@ allprojects {
   }
 
   spotless {
+    // TODO: Temporarily disable running spotlessCheck with the build task.
```

Comment:
Should the "TODO" be to remove this?
I'm not sure why this is being added.
In the description, should "with the build task" be "within the build task"?  I don't follow that.

All that said, if this is necessary temporarily, you can merge it.

Reply:
This just makes `./gradlew build`  not run `spotlessChecker`. 

I think I fixed the problem test failure in 
https://github.com/typetools/checker-framework/pull/5308. So it  and https://github.com/typetools/checker-framework/pull/5311 should be able to be merged soon.  Once those are merged, we can remove ` enforceCheck false`.